fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/pattern/Pattern.java

```diff
@@ -140,8 +139,8 @@ public boolean matchWithoutChild(OptExpression expression) {
         return getOpType().equals(expression.getOp().getOpType());
     }
 
-    private boolean isMultiJoin(OperatorType operatorType, int level) {
-        if (ALL_SCAN_TYPES.contains(operatorType) && level != 0) {
+    private boolean isMultiJoin(OperatorType operatorType) {
```

Comment:
you remove level ,and isMultiJoin will return true for scan. Do you consider to remove TF_MV_ONLY_SCAN_RULE and TF_MV_AGGREGATE_SCAN_RULE rules?